docs/release-notes/release-notes-0.18.0.md

```diff
@@ -65,6 +65,9 @@
 * [Properly handle un-acked updates for exhausted watchtower 
   sessions](https://github.com/lightningnetwork/lnd/pull/8233)
 
+* [Fix the removal of failed
+  channels](https://github.com/lightningnetwork/lnd/pull/8406).
+
```

Comment:
can you rebase once #8413  is merged and then add this to the 0.17.4 docs instead? 🙏

Reply:
waited for someone to create the 0.17.4 docs 😅